packaging/linux/deb/debroot.go

```diff
@@ -294,6 +301,22 @@ func fixupGenerators(spec *dalec.Spec, cfg *SourcePkgConfig) []byte {
 		fmt.Fprintf(buf, `test -n "$(go env GOMODCACHE)" || (GOPATH="$(go env GOPATH)"; mkdir -p "${GOPATH}/pkg" && ln -s "$(pwd)/%s" "${GOPATH}/pkg/mod")`, gomodsName)
 		// Above command does not have a newline due to quoting issues, so add that here.
 		fmt.Fprint(buf, "\n")
+		fmt.Fprintf(buf, "export GOMODCACHE=\"$(pwd)/%s\"\n", gomodsName)
+		fmt.Fprintf(buf, "export GOPROXY=\"file://$(pwd)/%s/cache/download\"\n", gomodsName)
+		fmt.Fprintln(buf, "export GOSUMDB=off")
+	}
+
+	script, err := dalec.GomodEditScript(spec)
```

Comment:
We don't want to do the edits here.
The edits should already be done and have patches produced.